tests_e2e/test_e2e.py

```diff
@@ -26,17 +27,27 @@ def createTrelloBoard(boardName):
 
 
 #Function to remove trello board
-def deleteTrelloBoard(boardId):
-    url='https://api.trello.com/1/boards/'+boardId
+def deleteTrelloBoard(boardId, key, token):
+    url='https://api.trello.com/1/boards/'+boardId+'?key='+key+'&token='+token
     res=requestJson("DELETE",url, {})
     return res
 
 @pytest.fixture(scope='module')
 def app_with_temp_board():
     # Create the new board & update the board id environment variable
-    board_id = (createTrelloBoard('Selenium_Test_Board'))["id"]
-    os.environ['trelloBoardID'] = board_id
+    board = (createTrelloBoard('Selenium_Test_Board'))
+    board_id = board["id"]
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
```

Comment:
Note that this currently happens _after_ you've tried to use these variables (in creating the board on the previous two lines)! If we shift this stage up to the top of this function then this test should be good to go 😄